scripts/build-images

```diff
@@ -51,14 +51,14 @@ xargs -n1 -t docker image pull --quiet << EOF > build/images-calico.txt
 EOF
 
 xargs -n1 -t docker image pull --quiet << EOF > build/images-vsphere.txt
-    ${REGISTRY}/rancher/mirrored-cloud-provider-vsphere-cpi-release-manager:v1.21.0
-    ${REGISTRY}/rancher/mirrored-cloud-provider-vsphere-csi-release-driver:v2.3.0
-    ${REGISTRY}/rancher/mirrored-cloud-provider-vsphere-csi-release-syncer:v2.3.0
-    ${REGISTRY}/rancher/mirrored-k8scsi-csi-node-driver-registrar:v2.1.0
-    ${REGISTRY}/rancher/mirrored-k8scsi-csi-resizer:v1.1.0
-    ${REGISTRY}/rancher/mirrored-k8scsi-livenessprobe:v2.2.0
-    ${REGISTRY}/rancher/mirrored-sig-storage-csi-attacher:v3.2.0
-    ${REGISTRY}/rancher/mirrored-sig-storage-csi-provisioner:v2.2.0
+    ${REGISTRY}/rancher/mirrored-cloud-provider-vsphere-cpi-release-manager:v1.23.0-alpha.1
```

Comment:
I believe we're using the 1.22 image, not 1.23 alpha? As per what's in rke2-charts at least:
https://github.com/rancher/rke2-charts/blob/main-source/packages/rancher-vsphere-cpi/generated-changes/patch/values.yaml.patch#L9